src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -25,22 +25,23 @@
  * @author Charles Sarrazin <charles@sarraz.in>
  * @author Kévin Dunglas <dunglas@gmail.com>
  * @author Antoine Bluchet <soyuka@gmail.com>
+ * @author Baptiste Meyer <baptiste.meyer@gmail.com>
```

Comment:
So that they can directly mail you instead of going through the issues... 
![mailapiplatforme](https://cloud.githubusercontent.com/assets/1321971/20341925/92a8dea0-abe8-11e6-9396-5e6485f570f2.png)